app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -1181,9 +1181,10 @@ export default {
   },
   async created() {
     if (this.submissionId && this.isDuplicate) {
-      //Run when make new submission from existing one called.
+      // Run when make new submission from existing one called. Get the
+      // published version of form, and then get the submission data.
+      await this.getFormSchema();
       await this.getFormData();
-      await this.getFormSchema(); //We need this to be called as well, because we need latest version of form
```

Comment:
The previous code loaded the data and then the form - for some reason this was causing the last item in a multiselect to be `undefined`.